README.md

```diff
@@ -34,15 +34,15 @@ OR
 ./gradlew run --debug-jvm # to just start a cluster that can be debugged
 ```
 
-The Elasticsearch server JVM will launch suspended and wait for a debugger to attach to `localhost:8000` before starting the Elasticsearch server.
+The Elasticsearch server JVM will connect to a debugger attached to `localhost:5005` before starting.
```

Comment:
Why do we want to have them to run on 2 different ports? Is there a case where they would need to be run at the same time?

Reply:
Yes we may need to run both simultaneously in the case when you want to debug integ tests.
Because integration tests run on different jvm and cluster on different jvm

Reply:
Not sure why you would do `./gradlew run --debug-jvm -Ddebug.es=1`. Do you mean running something like:
`./gradlew integTest -Ddebug.es=1 -Dtest.debug=1`?

Also I will change `debug.es` to be consistent with naming conventions.

Reply:
We might need this when we want to debug integ test all the way to the Elasticsearch code.